pkg/provisioner/cloudprovisioner.go

```diff
@@ -604,7 +605,8 @@ func (c *CloudProvisioner) ListSnapshots(
 		totalEntries = int(maxEntries)
 	}
 	entries := []v1alpha1.Snapshot{}
-	for count := 0; start < len(snapshots) && count < totalEntries; start++ {
+	count := 0
+	for ; start < len(snapshots) && count < totalEntries; start++ {
```

Comment:
This declaration change too?

Reply:
Oh, that was an artifact from an earlier attempt to fix the snapshot list `nextToken` calculation by adding the calculated `count` value to `start` on line 622 below. What I didn't realize at the time was that `start` was being incremented by the `for` loop. I reverted the addition, but forgot to revert this declaration. Will fix.

Reply:
Fixed.